platform/cli/src/index.js

```diff
@@ -94,4 +95,12 @@ program
     removeMode(packageName);
   });
 
+program
+  .command('list')
+  .description('List Added Extensions and Modes')
+  .action(() => {
+    const configPath = path.join(__dirname, '../../viewer/pluginConfig.json');
```

Comment:
Hmm, I think this is fine for now, but I think eventually the cli tool will be used in the viewer directory? I'm currently using all the other commands in the viewer directory like:

```node ../cli/src/index.js add-mode @ohif-test/mode-clock```

So maybe it should just check locally, and log an error if the user is not in an `@ohif/viewer` repository?

Reply:
Added